modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+// persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "persistent-storage-csi-vsphere"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:azure:
+endif::[]
+
+[id="csi-tp-enable_{context}"]
+= Enabling the {FeatureName} CSI driver operator
+To enable the {FeatureName} Container Storage Interface (CSI) driver operator, you must enable feature gates with the `TechPreviewNoUpgrade` feature set.
+
+.Procedure
+
+. Enable feature gates with the `TechPreviewNoUpgrade` feature set (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#[Enabling features using feature gates]).
++
+[IMPORTANT]
+====
+After turning Technology Preview features on using feature gates, they cannot be turned off and cluster upgrades are prevented.
+====
+. Verify the cluster operator storage:
++
+[source,terminal]
+----
+$ oc get co storage
+----
++
+[source, terminal]
+----
+NAME    VERSION                             AVAILABLE   PROGRESSING DEGRADED    SINCE
+storage 4.8.0-0.nightly-2021-04-30-201824   True        False       False       4h26m
+----
++
+* `AVAILABLE` should be "True".
+* `PROGRESSING` should be "False".
+* `DEGRADED` should be "False".
+
+. Verify the status of the pods in the `openshift-cluster-csi-drivers` namespace to verify that they are running:
```

Comment:
You have "verify" in the sentence twice. Suggest rewording to something like, "Check the status..."